cmd/config.go

```diff
@@ -116,6 +116,62 @@ func configShowCmd(ctx *config.Context) *cobra.Command {
 	return cmd
 }
 
+func configEditCmd(ctx *config.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "edit [path-name] [key] [value]",
+		Aliases: []string{"e"},
+		Short:   "Edit the config file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			home, err := cmd.Flags().GetString(flags.FlagHome)
+			if err != nil {
+				return err
+			}
+			cfgPath := path.Join(home, "config", "config.yaml")
+			if _, err := os.Stat(cfgPath); os.IsNotExist(err) {
+				if _, err := os.Stat(home); os.IsNotExist(err) {
+					return fmt.Errorf("home path does not exist: %s", home)
+				}
+				return fmt.Errorf("config does not exist: %s", cfgPath)
+			}
+			pathName := args[0]
+			key := args[1]
+			value := args[2]
+			configPath, err := ctx.Config.Paths.Get(pathName)
+			if err != nil {
+				return err
+			}
+			switch key {
+			case "client-id":
+				configPath.Src.ClientID = value
+				configPath.Dst.ClientID = value
```

Comment:
In general, src client id can be different from dst client id.
The same can be said about connection id, channel id, and port id.

Reply:
Add src/dst args
```
$RLY paths edit $PATH_NAME client-id src clinet-id-src
$RLY paths edit $PATH_NAME client-id dst clinet-id-dst
```